content/blog/youtrack-github.md

```diff
@@ -0,0 +1,36 @@
+---
+author: "Alex Hill"
+date: 2019-04-17
+title: Integrating YouTrack and GitHub workflows
+best: false
+---
+
+Like most technical teams we use an issue tracker to manage our work. Our tool of choice for
+this is [YouTrack](https://www.jetbrains.com/youtrack/). We also make extensive use of GitHub and to synchronise our 
+workflow across the two platforms we name our git branches with YouTrack ticket ids, so every pull request 
+corresponds to a ticket. This naming convention makes it easier to track work in progress across the two platforms, 
```

Comment:
Can you give an example (linking to a recent issue with a `mrc-xxx` or `VIMC-xxx` tag)

Reply:
It might also be worth mentioning why this naming system is useful over just plain github issues, for pieces of work that span multiple repositories we can use the same id

Reply:
actually we should avoid using the same id for multiple repos and really use subtasks/separate ids! otherwise the deploy tool and youtrack can be misleading (the PRs might be submitted at different times)